docs/docs/20-usage/21-multi-pipeline.md

```diff
@@ -1,11 +1,10 @@
+# Multi pipelines
 
-# Multi-pipeline builds
+> NOTE: This Feature is only available for Github & Gitea repositories. Follow [this](https://github.com/woodpecker-ci/woodpecker/issues/131) issue to support further development.
 
-**NOTE: Feature is only available for Github repositories. Follow [this](https://github.com/woodpecker-ci/woodpecker/issues/131) issue to support further development.**
+By default, Woodpecker looks for the pipeline definition in `.woodpecker.yml` in the project root.
 
-By default, Woodpecker looks for the pipeline definition in `.drone.yml` in the project root.
-
-The Multi-Pipeline feature allows the pipeline to be splitted to several files and placed in the `.drone/` folder
+The Multi-Pipeline feature allows the pipeline to be split to several files and placed in the `.woodpecker/` folder. Only `.yml` will we used and files in any subfolders like `.woodpecker/sub-folder/test.yml` will be ignored.
```

Comment:
Isn't this configurable from the UI on a per repo basis too?

Reply:
That's correct you can change `.woodpecker/` to some custom path in the project settings. 

```suggestion
The Multi-Pipeline feature allows the pipeline to be split into several files and placed in the `.woodpecker/` folder. Only `.yml` files will we used and files in any subfolders like `.woodpecker/sub-folder/test.yml` will be ignored. You can set some custom path like `.my-ci/pipelines/` instead of `.woodpecker/` in the [project settings](/docs/usage/project-settings). 
```